perfkitbenchmarker/linux_benchmarks/object_storage_service_benchmark.py

```diff
@@ -242,11 +194,13 @@
 DEFAULT = 'default'
 
 # This accounts for the overhead of running RemoteCommand() on a VM.
-MULTISTREAM_DELAY_PER_VM = 5.0
+MULTISTREAM_DELAY_PER_VM = 10.0
```

Comment:
Since we already have the units module, I made these into `units.second(...)` instead of plain floats. That documents what the units are, and it also means that PKB will automatically check that there are no unit conversion errors!